test/xsltproc_test.js

```diff
@@ -10,10 +10,16 @@ it("should return the correct TXT", function (done) {
       xml = '../xsltproc/test/fixtures/data.xml',
       opts = {
           "output": "test/fixtures/test.html",
-          "stringparam": {
-             "key": 'title',
-             "val": 'This is a single parameter passed as subtitle----anvidsahviulasdhvklasdbcuw'
-          }
+          "stringparam": [
```

Comment:
need add this in another case (it)

Reply:
You are absolutely right. 
I have set that test case as it was before and added the one for multiple string params.